mep/common/views.py

```diff
@@ -185,3 +189,51 @@ def render_facets(self, request, *args, **kwargs):
         '''Construct a JsonResponse based on the already-populated queryset
         data for the view.'''
         return JsonResponse(self.object_list.get_facets())
+
+
+# last modified view mixin adapted from ppa
+
+
+class SolrLastModifiedMixin(View):
+    """View mixin to add last modified headers based on Solr"""
+
+    # solr query filter for getting last modified date
+    solr_lastmodified_filters = {}   # by default, find all
+    #     filter_qs = ['item_type:work']
+
+    def get_solr_lastmodified_filters(self):
+        return self.solr_lastmodified_filters
+
+    def last_modified(self):
+        filter_qs = self.get_solr_lastmodified_filters()
+        sqs = SolrQuerySet().filter(**filter_qs) \
+            .order_by('-last_modified').only('last_modified')
+        try:
+            # Solr stores date in isoformat; convert to datetime
+            print(sqs[0])
```

Comment:
should this print statement still be here?

Reply:
ack no, thank you